sftp_to_ov.py

```
@@ -145,15 +150,23 @@ class MyCnOpts:
 		continue
 
 	for f in filteredFiles:
+		file_path = f'{sftp_directory}{f}'
+		processed_file_path = f'{sftp_directory}{PROCESSED_FOLDER_NAME}/{f}'
+
 		Message(f)
 		try:
-			sftp.get(parameters['SFTP']['Directory']+f, preserve_mtime=True)
+			sftp.get(file_path, preserve_mtime=True)
 		except:
 			Message(sys.exc_info)
 			quit(1) # process files on next fun.  Error on getting file usually because file is still being written to.
 
 		if runAndWaitForImport(f, parameters["IMPORTS"][imp]["impspec"], parameters["IMPORTS"][imp]["action"], parameters["IMPORTS"][imp]["maxRuntimeInMinutes"]):
-			sftp.rename(parameters['SFTP']['Directory']+f,parameters['SFTP']['Directory']+'processed/'+f)
+			try:
+				sftp.remove(processed_file_path)
+			except FileNotFoundError:
+				pass
```

Add a comment about why you ignore exception. Also, it is usually faster to check if a file exists and only if it is - remove it

---

This exception is needed because the file may not be in the processed folder. And this implementation is made in sftp_to_onevizion_import.py of this module, I just thought to do the same, to make it easier to refactor this module later.
But okay, I changed it to check if a file exists.